examples/nextjs/components/ProductCard/ProductCard.js

```diff
@@ -28,6 +27,16 @@ const ProductCard = ({
   showDescription = false,
   constrainImages = true,
   isPDP = false,
+  dimensions = {
```

Comment:
This feels a bit too specific to be a prop. Should we not just pass in the dimensions? We could make the dimensions for PDPs & PLPs constants that are passed in when used, or even just referenced here in the component when doing the `isPDP` check. Passing it in as an object and having the `plp` and `pdp` properties feels like it's not generic enough to be used as an external-facing component prop.

Reply:
Agreed. Changed to have the `width` and `height` values passed directly.